plasmapy/diagnostics/thomson.py

```diff
@@ -68,62 +69,62 @@ def spectral_density_lite(
     The :term:`lite-function` version of
     `~plasmapy.diagnostics.thomson.spectral_density`.  Performs the same
     thermal speed calculations as
-    `~plasmapy.diagnostics.thomson.spectral_density`, but is intended for
-    computational use and, thus, has data conditioning safeguards
+    `~plasmapy.diagnostics.thomson.spectral_density`, but is intended
+    for computational use and thus has data conditioning safeguards
     removed.
 
     Parameters
     ----------
-    wavelengths : `~numpy.ndarray`, shape (Nwavelengths,)
-        Array of wavelengths in meters over which the spectral density function
-        will be calculated.
+    wavelengths : (Nλ,) `~numpy.ndarray`
+        The wavelengths in meters over which the spectral density
+        function will be calculated.
 
     probe_wavelength : real number
         Wavelength of the probe laser in meters.
 
-    n : `~astropy.units.Quantity`
-        Total combined number density of all electron populations.
-        (in m\ :sup:`-3`)
+    n : `~numpy.ndarray`
+        Total combined number density of all electron populations in
+        m\ :sup:`-3`\ .
 
-    T_e : `~numpy.ndarray`, shape (Ne, )
-        Temperature of each electron component in kelvin. Shape (Ne, ) must be
-        equal to the number of electron populations Ne.
+    T_e : (Ne,) `~numpy.ndarray`
+        Temperature of each electron population in kelvin, where Ne is
+        the number of electron populations.
```

Comment:
In the docstrings here, "population" and "component" were being used interchangeably, and I was thinking it'd be helpful to use one or the other consistently.  I tentatively went with population only because it implies multiple things, but I'd be happy to go with either.

Reply:
Agreed - they are pretty much interchangeable and I think your decision here is good